src/protocol/DocumentServer.js

```diff
@@ -222,6 +223,33 @@ define('polymer-designer/protocol/DocumentServer', [
       });
     }
 
+    backspaceText(request) {
+      let text = request.message.text;
+      let node = document.createTextNode(text);
```

Comment:
I tried it without it and it does not work. Cursor does not move when backspacing.